### PR TITLE
HDDS-1216. Change name of ozoneManager service in docker compose file…

### PR DESCRIPTION
**Pull request by ajayydv**
…s to om. Contributed by Ajay Kumar.

### CODE REVIEW

File: hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot [hunk 1]
```diff
@@ -107,5 +140,16 @@ Run ozoneFS tests
...
     ${rc}  ${result} =  Run And Return Rc And Output        ozone fs -ls o3fs://abcde.pqrs/
                         Should Be Equal As Integers     ${rc}                1
-                        Should contain    ${result}         VOLUME_NOT_FOUND
+                        Should contain    ${result}         not found
+
+
+Secure S3 test Failure
+    Run Keyword         Install aws cli
+    ${rc}  ${result} =  Run And Return Rc And Output  aws s3api --endpoint-url ${ENDPOINT_URL} create-bucket --bucket bucket-test123
+    Should Be True	${rc} > 0
```
Comment (hadoop-yetus) [inline hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot]: whitespace:tabs in line

File: hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot [hunk 2]
```diff
@@ -16,14 +16,44 @@
...
+    Run Keyword if     '${rc}' == '0'              Install aws cli s3 debian
+    ${rc}              ${output} =                 Run And Return Rc And Output           yum --help
+    Run Keyword if     '${rc}' == '0'              Install aws cli s3 centos
+
+Setup credentials
+    ${hostname}=        Execute                    hostname
+    Execute             kinit -k testuser/${hostname}@EXAMPLE.COM -t /etc/security/keytabs/testuser.keytab
+    ${result} =         Execute                    ozone sh s3 getsecret
+    ${accessKey} =      Get Regexp Matches         ${result}     (?<=awsAccessKey=).*
+    ${secret} =         Get Regexp Matches	       ${result}     (?<=awsSecret=).*
```
Comment (hadoop-yetus) [inline hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot]: whitespace:tabs in line

File: hadoop-ozone/dist/src/main/smoketest/test.sh [hunk 3]
```diff
@@ -78,7 +78,7 @@ execute_tests(){
      TITLE="Ozone $TEST tests with $COMPOSE_DIR cluster"
      set +e
      OUTPUT_NAME="$COMPOSE_DIR-${TEST//\//_}"
-	  docker-compose -f "$COMPOSE_FILE" exec -T ozoneManager python -m robot --log NONE --report NONE "${OZONE_ROBOT_OPTS[@]}" --output "smoketest/$RESULT_DIR/robot-$OUTPUT_NAME.xml" --logtitle "$TITLE" --reporttitle "$TITLE" "smoketest/$TEST"
+	  docker-compose -f "$COMPOSE_FILE" exec -T om python -m robot --log NONE --report NONE "${OZONE_ROBOT_OPTS[@]}" --output "smoketest/$RESULT_DIR/robot-$OUTPUT_NAME.xml" --logtitle "$TITLE" --reporttitle "$TITLE" "smoketest/$TEST"
```
Comment (hadoop-yetus) [inline hadoop-ozone/dist/src/main/smoketest/test.sh]: whitespace:tabs in line

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 20 | Docker mode activated. |
||| _ Prechecks _ |
| 0 | yamllint | 0 | yamllint was not available. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1091 | trunk passed |
| -1 | compile | 24 | dist in trunk failed. |
| -1 | mvnsite | 26 | dist in trunk failed. |
| +1 | shadedclient | 706 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 20 | trunk passed |
||| _ Patch Compile Tests _ |
| -1 | mvninstall | 20 | dist in the patch failed. |
| -1 | compile | 20 | dist in the patch failed. |
| -1 | javac | 20 | dist in the patch failed. |
| -1 | mvnsite | 21 | dist in the patch failed. |
| +1 | shellcheck | 1 | There were no new shellcheck issues. |
| +1 | shelldocs | 15 | There were no new shelldocs issues. |
| -1 | whitespace | 0 | The patch 3  line(s) with tabs. |
| +1 | shadedclient | 838 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 17 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 22 | dist in the patch failed. |
| +1 | asflicense | 27 | The patch does not generate ASF License warnings. |
| | | 3001 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-553/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/553 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  yamllint  shellcheck  shelldocs  |
| uname | Linux 1ea77581ba55 4.4.0-139-generic #165~14.04.1-Ubuntu SMP Wed Oct 31 10:55:11 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / cb0fa0c |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/1/artifact/out/branch-compile-hadoop-ozone_dist.txt |
| mvnsite | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/1/artifact/out/branch-mvnsite-hadoop-ozone_dist.txt |
| shellcheck | v0.4.6 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/1/artifact/out/patch-mvninstall-hadoop-ozone_dist.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/1/artifact/out/patch-compile-hadoop-ozone_dist.txt |
| javac | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/1/artifact/out/patch-compile-hadoop-ozone_dist.txt |
| mvnsite | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/1/artifact/out/patch-mvnsite-hadoop-ozone_dist.txt |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/1/artifact/out/whitespace-tabs.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/1/artifact/out/patch-unit-hadoop-ozone_dist.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/1/testReport/ |
| Max. process+thread count | 340 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/dist U: hadoop-ozone/dist |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot (hunk 2 above)
Comment (bharatviswa504) [reply hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot]: Can we remove this change from here? As this Jira purpose is not for this.

File: hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot [hunk 4]
```diff
@@ -107,5 +140,16 @@ Run ozoneFS tests
...
+
+Secure S3 test Failure
+    Run Keyword         Install aws cli
+    ${rc}  ${result} =  Run And Return Rc And Output  aws s3api --endpoint-url ${ENDPOINT_URL} create-bucket --bucket bucket-test123
+    Should Be True	${rc} > 0
+
+Secure S3 test Success
+    Run Keyword         Setup credentials
+    ${output} =         Execute          aws s3api --endpoint-url ${ENDPOINT_URL} create-bucket --bucket bucket-test123
+
```
Comment (bharatviswa504) [inline hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot]: Can we remove S3 related changes from this?

File: hadoop-ozone/dist/src/main/compose/ozone/docker-compose.yaml [hunk 5]
```diff
@@ -26,15 +26,15 @@ services:
...
+   om:
       image: apache/hadoop-runner
       privileged: true #required by the profiler
       volumes:
          - ../..:/opt/hadoop
       ports:
          - 9874:9874
       environment:
-         ENSURE_OM_INITIALIZED: /data/metadata/ozoneManager/current/VERSION
+         ENSURE_OM_INITIALIZED: /data/metadata/om/current/VERSION
```
Comment (bharatviswa504) [inline hadoop-ozone/dist/src/main/compose/ozone/docker-compose.yaml]: Here we have changed to om, will be this be same as service name when path is created?

File: hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot (hunk 2 above)
Comment (arp7) [reply hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot]: +1 to move it to a separate Jira. Let's keep only ozoneManager -> om refactoring changes in this patch.

File: hadoop-ozone/dist/src/main/compose/ozone/docker-compose.yaml (hunk 5 above)
Comment (ajayydv) [reply hadoop-ozone/dist/src/main/compose/ozone/docker-compose.yaml]: changed this for consistency purpose. IMO path can be independent of service name.

File: hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot (hunk 4 above)
Comment (ajayydv) [reply hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot]: removed s3 related changes, HDDS-1043 will have them.

File: hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot (hunk 2 above)
Comment (ajayydv) [reply hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot]: removed s3 related changes, HDDS-1043 will have them.

Comment (elek) [reply hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot]: Still we have a few of them (commonawslib, docker-config)

File: hadoop-ozone/dist/src/main/smoketest/test.sh (hunk 3 above)
Comment (hadoop-yetus) [inline hadoop-ozone/dist/src/main/smoketest/test.sh]: whitespace:tabs in line

Comment (hadoop-yetus) [inline hadoop-ozone/dist/src/main/smoketest/test.sh]: whitespace:tabs in line

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 26 | Docker mode activated. |
||| _ Prechecks _ |
| 0 | yamllint | 0 | yamllint was not available. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1427 | trunk passed |
| +1 | compile | 65 | trunk passed |
| +1 | mvnsite | 27 | trunk passed |
| +1 | shadedclient | 687 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 22 | trunk passed |
||| _ Patch Compile Tests _ |
| -1 | mvninstall | 20 | dist in the patch failed. |
| +1 | compile | 19 | the patch passed |
| +1 | javac | 19 | the patch passed |
| +1 | mvnsite | 21 | the patch passed |
| +1 | shellcheck | 1 | There were no new shellcheck issues. |
| +1 | shelldocs | 15 | The patch generated 0 new + 104 unchanged - 136 fixed = 104 total (was 240) |
| -1 | whitespace | 4 | The patch has 75 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply |
| -1 | whitespace | 5 | The patch 19850  line(s) with tabs. |
| +1 | shadedclient | 996 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 18 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 22 | dist in the patch passed. |
| +1 | asflicense | 28 | The patch does not generate ASF License warnings. |
| | | 3550 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-553/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/553 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  yamllint  shellcheck  shelldocs  |
| uname | Linux ddc54f9835cf 4.4.0-138-generic #164~14.04.1-Ubuntu SMP Fri Oct 5 08:56:16 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 7f636b4 |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| shellcheck | v0.4.6 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/2/artifact/out/patch-mvninstall-hadoop-ozone_dist.txt |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/2/artifact/out/whitespace-eol.txt |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/2/artifact/out/whitespace-tabs.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/2/testReport/ |
| Max. process+thread count | 307 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/dist U: hadoop-ozone/dist |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/2/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 25 | Docker mode activated. |
||| _ Prechecks _ |
| 0 | yamllint | 0 | yamllint was not available. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1098 | trunk passed |
| +1 | compile | 71 | trunk passed |
| +1 | mvnsite | 28 | trunk passed |
| +1 | shadedclient | 644 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 23 | trunk passed |
||| _ Patch Compile Tests _ |
| -1 | mvninstall | 19 | dist in the patch failed. |
| +1 | compile | 19 | the patch passed |
| +1 | javac | 19 | the patch passed |
| +1 | mvnsite | 22 | the patch passed |
| +1 | shellcheck | 1 | There were no new shellcheck issues. |
| +1 | shelldocs | 17 | The patch generated 0 new + 104 unchanged - 136 fixed = 104 total (was 240) |
| -1 | whitespace | 4 | The patch has 75 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply |
| -1 | whitespace | 5 | The patch 19850  line(s) with tabs. |
| +1 | shadedclient | 993 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 18 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 24 | dist in the patch passed. |
| +1 | asflicense | 31 | The patch does not generate ASF License warnings. |
| | | 3185 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-553/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/553 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  yamllint  shellcheck  shelldocs  |
| uname | Linux da1508afe25b 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 7f636b4 |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| shellcheck | v0.4.6 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/3/artifact/out/patch-mvninstall-hadoop-ozone_dist.txt |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/3/artifact/out/whitespace-eol.txt |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/3/artifact/out/whitespace-tabs.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/3/testReport/ |
| Max. process+thread count | 414 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/dist U: hadoop-ozone/dist |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/3/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot (hunk 2 above)
Comment (ajayydv) [reply hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot]: @elek  thanks for catching that, removed from both files (commonawslib, docker-config).

File: hadoop-ozone/dist/src/main/smoketest/test.sh (hunk 3 above)
Comment (ajayydv) [reply hadoop-ozone/dist/src/main/smoketest/test.sh]: not sure what is the problem here, change involves just rename.

Comment (hadoop-yetus) [inline hadoop-ozone/dist/src/main/smoketest/test.sh]: whitespace:tabs in line

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 26 | Docker mode activated. |
||| _ Prechecks _ |
| 0 | yamllint | 0 | yamllint was not available. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1124 | trunk passed |
| +1 | compile | 87 | trunk passed |
| +1 | mvnsite | 32 | trunk passed |
| +1 | shadedclient | 614 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 25 | trunk passed |
||| _ Patch Compile Tests _ |
| -1 | mvninstall | 19 | dist in the patch failed. |
| +1 | compile | 19 | the patch passed |
| +1 | javac | 19 | the patch passed |
| +1 | mvnsite | 20 | the patch passed |
| +1 | shellcheck | 1 | There were no new shellcheck issues. |
| +1 | shelldocs | 18 | The patch generated 0 new + 104 unchanged - 136 fixed = 104 total (was 240) |
| -1 | whitespace | 3 | The patch has 75 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply |
| -1 | whitespace | 4 | The patch 19850  line(s) with tabs. |
| +1 | shadedclient | 894 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 15 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 22 | dist in the patch passed. |
| +1 | asflicense | 28 | The patch does not generate ASF License warnings. |
| | | 3107 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-553/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/553 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  yamllint  shellcheck  shelldocs  |
| uname | Linux 4db865cc78cf 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 1ad5bfc |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| shellcheck | v0.4.6 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/4/artifact/out/patch-mvninstall-hadoop-ozone_dist.txt |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/4/artifact/out/whitespace-eol.txt |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/4/artifact/out/whitespace-tabs.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/4/testReport/ |
| Max. process+thread count | 446 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/dist U: hadoop-ozone/dist |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-553/4/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-ozone/dist/src/main/smoketest/test.sh (hunk 3 above)
Comment (elek) [reply hadoop-ozone/dist/src/main/smoketest/test.sh]: IMHO, you modified a line which already contained a tab.

**Comment by elek** [comment]
Merged. Thanks you very much @ajayydv the contribution. I am very happy that we started to use the shorter names.

**Pull request closed without merging**